index.scss

```diff
@@ -11,84 +11,89 @@
   }
 }
 
-.v2-alert {
-  padding: 20px 20px 20px 86px;
+.alert {
+  display: inline-block;
   position: relative;
-  min-height: 50px;
-  font-size: 16px;
+  min-height: 52px;
   overflow: hidden;
-  background-color: #fff;
-  border-radius: 3px;
 }
 
-.v2-alert::after {
+.alert::after {
   /* use nib-styles/v2-icons as font to get icons */
   font: normal normal normal 14px/1 'nibdings';
   text-rendering: auto;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  font-size: 40px;
+  font-size: 27px;
   position: absolute;
-  padding-top: 10px;
-  top: 15px;
-  bottom: 15px;
-  left: 0;
-  width: 65px;
-  color: #fff;
+  padding-top: 16px;
+  top: -4px;
+  left: -4px;
+  width: 60px;
+  min-height: 100%;
   text-align: center;
-  animation: v2-alert-pop-out 0.6s cubic-bezier(0.19, 1, 0.22, 1);;
 }
 
-.v2-alert--error {
-  background-color: #df6666;
-  color: #fff;
+.alert-msg--single {
+  font-size: 18pt;
+  font-family: 'Roboto';
+  font-weight: lighter;
+  padding: 0px 12px 0px 72px;
+  min-height: 52px;
+  line-height: 52px;
 }
 
-.v2-alert--error::after {
-  content: "\E004";
-  color: #fff;
-  border-right: 2px solid #fff;
+.alert--title {
+  font-size: 18px;
+  font-family: 'Roboto';
+  font-weight: bold;;
+  padding: 12px 12px 12px 72px;
 }
 
-.v2-alert--info {
-  background-color: #55a2df;
-  color: #fff;
+.alert--message {
+  font-size: 14px;
+  font-family: 'open sans';
+  font-weight: lighter;
+  padding: 0px 12px 12px 72px;
 }
 
-.v2-alert--info::after {
-  content: "\E00D";
-  color: #fff;
-  border-right: 2px solid #fff;
-}
 
-.v2-alert--success {
-  background-color: #009623;
-  color: #fff;
+.alert--error {
+  border: 4px solid #E74C3C;
 }
 
-.v2-alert--success::after{
-  content: "\E01C";
+.alert--error::after {
+  content: "\E002";
```

Comment:
I was able to use the mixins but it wasn't playing nice with the other styles; in particular it was overriding the white background for the text.